prod/westeurope/linux/appbackendl1/app_service/terragrunt.hcl

```
@@ -168,10 +168,16 @@ inputs = {
     BONUS_API_URL = "http://${dependency.functions_bonus.outputs.default_hostname}/api/v1"
     CGN_API_URL   = "http://${dependency.functions_cgn.outputs.default_hostname}/api/v1"
 
+    // THIRD PARTY APIS
+    HERE_AUTOCOMPLETE_API_URL = "https://autocomplete.search.hereapi.com"
```

```suggestion
    GEO_HERE_AUTOCOMPLETE_API_URL = "https://autocomplete.search.hereapi.com"
```
can we add GEO_* prefix for these vars?

---

Yep, I'll update them in io-backend :)

---

thanks @AleDore insert also this reference under // THIRD PARTY APIS https://www.here.com/